ethtxmanager/ethtxmanager.go

```diff
@@ -6,14 +6,16 @@ package ethtxmanager
 
 import (
 	"context"
+	"errors"
 	"fmt"
 	"math/big"
-	"strings"
 	"time"
 
 	ethmanTypes "github.com/0xPolygonHermez/zkevm-node/etherman/types"
 	"github.com/0xPolygonHermez/zkevm-node/log"
 	"github.com/0xPolygonHermez/zkevm-node/proverclient/pb"
+	"github.com/0xPolygonHermez/zkevm-node/state/runtime"
+	"github.com/0xPolygonHermez/zkevm-node/test/operations"
```

Comment:
It is not a good practice import test packages. I know it is imported in the etherman and it's ok for this PR. But we should create an issue to fix it, just to be sure we don't missed it. Can you do it?

Reply:
Totally agree! I was wondering why we have this functionality in the test package when it is used in a lot of other non-test files. But just for this PR will leave it.